doorpi.py

```diff
@@ -187,7 +188,9 @@ def __init__(self, *args, **kwargs):
         # TODO: Remove development hack (maybe)
         global emulation
         if not emulation:
-            DoorSocketHandler.door = DigitalOutputDevice(int(Application.config('gpio.open')))
+            DoorSocketHandler.door = Door("RindusDoor",
```

Comment:
Name of the door could be found by Application.config('door.name'), but I think name is not important here as one DoorPI only handles one door installation. Multiple doors should be handled by a door-manager.

Reply:
sure.